### ISSUE TITLE
Invalid private key can be generated.

### ISSUE THREAD

**Issue by AmritKumar** [Zilliqa/Zilliqa-JavaScript-Library#92]
**Describe the bug**
In `Line 45` in `packages/zilliqa-js-account/src/wallet.ts`, the following code is used to generate a private key:
```
const privateKey = zcrypto.randomBytes(32);
```
It seems that there is no check on the value of `privateKey`. Private keys in Schnorr must lie between `[1,..., n-1]`, where `n` is the curve order and n < 2**256. In fact, this bug is serious in the sense that one may be able to generate an invalid wallet address (with low probability) and receive funds there but the funds can never be spent as the signature module in Schnorr (`packages/zilliqa-js-crypto/src/schnorr.ts`) indeed checks whether the private key lies between `[1,..., n-1]`.
```
  if (privKey.isZero()) {
    throw new Error('Bad private key.');
  }

  if (privKey.gte(curve.n)) {
    throw new Error('Bad private key.');
}
```

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by iantanwx** [comment]
Fixed by 1c3beb7ed9c0b39eea5de09c7ca062ffa2107df7

**Comment by Acwstar** [comment]
How to fix bad private key 
I don't know progam github
